parser/sql/dialect/oracle/src/main/antlr4/imports/oracle/PLSQL.g4

```diff
@@ -314,6 +321,13 @@ ifStatement
     END IF SEMI_
     ;
 
+modifyingStatement
+    : IF modifyingExpression THEN statement+
```

Comment:
Can you combine these subrules into one line? Line breaks are required only when the subrule exceeds 5.

Reply:
Okay, I'll make the changes right away.

Reply:
I have finished the modifications now.